### PR TITLE
VideoPress: document actions and filters.

### PR DESCRIPTION
**Pull request by jeherve**

### CODE REVIEW

File: modules/videopress/class.videopress-player.php [hunk 1]
```diff
@@ -597,7 +597,16 @@ private function get_flash_parameters() {
 		if ( ! isset( $this->video->players->swf->params ) )
 			return array();
 		else
-			return self::esc_flash_params( apply_filters( 'video_flash_params', (array) $this->video->players->swf->params, 10, 1 ) );
+			return self::esc_flash_params(
+				/**
+				 * Filters the Flash parameters of the VideoPress player.
```
Comment (kraftbj) [inline modules/videopress/class.videopress-player.php]: Should we note that a new version of VideoPress is on deck and this filter's behavior will be changed/removed in an upcoming release?

File: modules/videopress/shortcode.php [hunk 2]
```diff
@@ -63,6 +63,13 @@ public function shortcode_callback( $attr, $content = '' ) {
 		if ( ( $attr['width'] % 2 ) === 1 )
 			$attr['width']--;
 
+		/**
+		 * Filter the default VideoPress shortcode options.
+		 *
+		 * @since 2.5.0
+		 *
+		 * @param array $args Array of VideoPress shortcode options.
```
Comment (kraftbj) [inline modules/videopress/shortcode.php]: Same here, without being aware of new VP shortcodes operate any differently.

File: modules/videopress/class.videopress-player.php (hunk 1 above)
Comment (jeherve) [inline modules/videopress/class.videopress-player.php]: I assume we'll just add the hooks to [the list of deprecated hooks](https://github.com/Automattic/jetpack/blob/95e01888e92029cef3db6afb8275f75e0a9bf1c4/class.jetpack.php#L5570) when we remove them? We could add them now if we knew for sure that they'll be removed in 3.7.

Comment (kraftbj) [inline modules/videopress/class.videopress-player.php]: Ah yes. Forgot about that. That works for me. I wouldn't want to add them now until we're sure what VP will look like in JP.

**Comment by samhotchkiss** [comment]
Since the VideoPress update should be shipping in this release, I'm holding off on this and punting over to @zinigor to consider alongside his VP work

**Pull request merged**